docker/Dockerfile

```diff
@@ -134,23 +133,16 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get -y install \
     libglu1 \
     libxcursor-dev \
-    libxinerama1 && \
-    apt-get -y install \
-    python3-lxml && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    libxinerama1
```

Comment:
Why were the last two lines removed? I gather that they were there to reduce the docker image size.